config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  resources :users
+  resources :posts
+  root to: 'users#index'
+  get 'posts/index'
+  get 'posts/show'
+  get 'users/index'
+  get 'users/show'
```

Comment:
- [ ] Posts routes are not properly configured as requested in the project requirements and also you created some routes more than once. Please configure your routes correctly to match the given `rails routes` command output in the project requirement.

![image](https://github.com/ReemMohamedAbdelfatah/rails-blog-app/assets/68796232/d32fd638-d8fa-4320-91c4-0b4e618a9aae)
```suggestion
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end
  
root to: 'users#index'
```

Reply:
I used "rails generate controller Users index show" is it considered scaffold?